src/bindings/python/tests/test_torchvision_to_ov/test_preprocessor.py

```diff
@@ -95,7 +95,7 @@ def test_convertimagedtype():
         ],
     )
     torch_result, ov_result = _infer_pipelines(test_input, preprocess_pipeline)
-    assert np.max(np.absolute(torch_result - ov_result)) < 2e-04
+    assert np.max(np.absolute(torch_result - ov_result)) < 3e-04
```

Comment:
is it OV or TorchVision difference?

Reply:
At this point it's impossible to tell, it's just a difference between the two. In order to know which is "more" correct we would need some reference result.